android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -100,6 +101,11 @@ public CodePush(String deploymentKey, Activity mainActivity, boolean isDebugMode
         }
 
         initializeUpdateAfterRestart();
+        if (currentInstance != null) {
+            CodePushUtils.log("More than one CodePush instances has been initialized, please use the non-static method codePush.getBundleUrlInternal() to get the correct bundleURL for a particular instance.");
```

Comment:
"More than one CodePush instance has been initialized.  Please use the instance method codePush.getBundleUrlInternal() to get the correct bundleURL for a particular instance."